include/tsid/formulations/inverse-dynamics-formulation-acc-force.hpp

```diff
@@ -31,16 +31,15 @@ namespace tsid
   {
   public:
     EIGEN_MAKE_ALIGNED_OPERATOR_NEW
-    
+
     double time_start;
     double time_end;
-    double fMax_start;  /// max normal force at time time_start
-    double fMax_end;    /// max normal force at time time_end
+    double fMax_start; /// max normal force at time time_start
+    double fMax_end;   /// max normal force at time time_end
```

Comment:
In this file there are cosmetic changes that have nothing to do with the PR and should be removed.